uavcan/__init__.py

```diff
@@ -120,6 +120,10 @@ def load_dsdl(*paths, **args):
         if not args.get("exclude_dist", None):
             dsdl_path = pkg_resources.resource_filename(__name__, "dsdl_files")  # @UndefinedVariable
             paths = [os.path.join(dsdl_path, "uavcan")] + paths
+            custom_path = os.path.join(os.path.expanduser("~"),"uavcan_custom_types")
+            if os.path.isdir(custom_path):
+                paths = [f for f in [os.path.join(custom_path,f) for f in os.listdir(custom_path)] 
+                            if os.path.isdir(f)] + paths
```

Comment:
You're appending the custom entries before the default path, why? They should go after, use `+=`.

Please adhere to the coding conventions (PEP8), you need to fix spaces after commas and new line alignment before the `if`.

Maybe we could improve clarity by naming the directory `uavcan_vendor_specific_types`? "Vendor specific data type" is a term that is widely used in the specification.